src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -72,13 +69,39 @@ In the **Regions & nodes** section, select the number of nodes.
 
 Currently, you can add a maximum of 150 nodes to your cluster. For larger configurations, [contact us](https://support.cockroachlabs.com/hc/requests/new).
 
-## Step 5. Select the hardware per node
+Click **Next: Capacity**.
+
+## Step 4. Enable VPC Peering (optional)
```

Comment:
Since this step comes right after "Click Next: Capacity", it's unclear that "Enable VPC Peering" is under the "Cloud & Regions" section as well. It sounds like this is the first step on the Capacity page.

Reply:
Good catch!